examples/tests/sdk_tests/bucket/signedUrl.w

```diff
@@ -0,0 +1,14 @@
+bring cloud;
```

Comment:
Please add negative test as well - for instance, what happens when there is no url.